alerting/provider/pushover/pushover.go

```diff
@@ -34,6 +34,10 @@ type AlertProvider struct {
 	// default: 0
 	Priority int `yaml:"priority,omitempty"`
 
+	// Priority of resolved messages, ranging from -2 (very low) to 2 (Emergency)
+	// default: Priority
```

Comment:
Should we default 0 here?

Reply:
What's the default if you don't specify it in pushover? If it's 0, then yes, we should use 0

Reply:
Isn't 0 already the default?

Reply:
Yes but my worry is that this is kind of a breaking change. Meaning previously if someone set the priority to 2, the resolved priority would be the same. With this PR the priority would still be 2 but the resolved priority would be 0 unless the person updated the resolved priority to be 2 as well.

Reply:
@TwiN are we fine with this being a "breaking" change and just assigning `resolved-priority` to a default of `0` instead of trying to hack in `priority == resolved-priority` if `resolved-priority` is not set?

Reply:
Ah, I see your point. I don't think the priority aspect of pushover is worth worrying about as far as breaking changes go, especially since this is just the priority for the resolve notification. I think it's fine if we have it default to 0, despite the fact that it technically changes the previous behavior.

Reply:
Alrighty, everything should be good to go then